src/Pkg.jl

```diff
@@ -273,6 +270,41 @@ const develop = API.develop
 #TODO: Will probably be deprecated for something in PkgDev
 const generate = API.generate
 
+"""
+    Pkg.dependencies()::Dict{UUID, PackageInfo}
+
+Query the dependecy graph.
+The result is a `Dict` that maps a package UUID to a `PackageInfo` struct representing the dependency (a package).
+
+PackageInfo fields:
+
+| `field`        | `Description`                                              |
+|:---------------|:-----------------------------------------------------------|
+| `name`         | The name of the package                                    |
+| `version`      | The version of the package (this is `Nothing` for stdlibs) |
+| `isdeveloped`  | Whether a package is directly tracking a directory         |
+| `ispinned`     | Whether a package is pinned                                |
+| `source`       | The directory containing the source code for that package  |
+| `dependencies` | The dependencies of that package as a vector of UUIDs      |
+"""
+const dependencies = API.dependencies
+
+"""
+    Pkg.project()::ProjectInfo
```

Comment:
I wonder if `ProjectInfo(::UUID)` would be a good constructor? Otherwise it is a bit hard to do something with the `.dependencies` field.